.vscode/launch.json

```diff
@@ -0,0 +1,29 @@
+{
+    // Use IntelliSense to learn about possible Node.js debug attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
```

Comment:
I think this comment can be removed – it's invalid JSON, and also doesn't look like it's content is crucial to kicking off a new project ;)